components/ContractOwner/ContractOwner.tsx

```diff
@@ -8,44 +8,51 @@ import { INTERFACE_IDS } from '@lukso/lsp-smart-contracts';
 import AddressButtons from '../AddressButtons';
 
 type Props = {
-  UPAddress: string;
+  contractAddress: string;
 };
 
 enum ownerTypeEnum {
-  KeyManager = 'Key Manager',
-  SmartContract = 'Smart Contract',
-  EOA = 'EOA',
+  EOA = '🗝️ EOA',
+  SmartContract = '📄 Smart Contract',
+  ERC725Account = '🆙 ERC725 Account',
+  KeyManager = '🔐 Key Manager',
```

Comment:
this enum could prob go in a more global "types" file so it can be reused elsewhere.